scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/kms/README.md

```diff
@@ -0,0 +1,34 @@
+```mermaid
+classDiagram
+    KeyManagementService ..> SimpleKeyManagementService
```

Comment:
The parent-child class relationship is represented as solid line from child to parent in UML, namely **KMS<|--SimpleKMS** rather than **KMS..>SimpleKMS**